CHANGELOG.md

```diff
@@ -29,13 +29,29 @@ This release has an [MSRV][] of 1.85.
 
 ### Removed
 
-- Breaking: `Renderer::render_to_surface` has been removed. ([#803][] by [@DJMcNab][])
+- Breaking: `Renderer::render_to_surface` has been removed. ([#803][] by [@DJMcNab][])  
```

Comment:
Is this whitespace suffix intentional?

```suggestion
- Breaking: `Renderer::render_to_surface` has been removed. ([#803][] by [@DJMcNab][])
```

Reply:
Aha, yes. Good to know this whitespace achieves that. I've used backslash for the same effect before.